planning/rtc_replayer/launch/rtc_replayer.launch.xml

```diff
@@ -0,0 +1,7 @@
+<launch>
+  <arg name="param_path" default="$(find-pkg-share rtc_auto_mode_manager)/config/rtc_auto_mode_manager.param.yaml"/>
+
```

Comment:
this param file does not seem to exist in odaiba3

Reply:
this was added with https://github.com/autowarefoundation/autoware.universe/pull/1541 to main